### DISCUSSION TITLE
#34 togETHer biweekly meeting

### DISCUSSION THREAD

**Discussion by bobjiang** [tog-ETH-er/togETHer#9] [General]
Agenda:

- Community Updates
- Fireside discussion
- Hackthon

会议主题：togETHer
会议时间： 21:00-22:00
重复周期：每两周 (周一)

点击链接入会，或添加至会议列表：
https://meeting.tencent.com/s/LxNajKWJw4RV

会议 ID：891 5098 6775

**Comment by winkybupt** [comment]
## 原语里弄
1. 梳理了部分高质量信息源并制作了feed，有兴趣的可以联系测试。
2. 办了一场读书会。

**Comment by Riely007** [comment]
### **以太坊翻译小组** 

1. 继续以太坊官网的翻译，目前进度 95%  - ethereum.org 的 discord 里有专门的中文翻译小组频道，[感兴趣的可以加入](https://discord.gg/2uggQg2v)

**Comment by bobjiang** [comment]
- Gitcoin Grant Round 13 - Mar 9 - 24
- Claim your matching fund (GR12)
- Add tags for your grants - tags@gitcoin.co
- ENS tags for grant
- ENS ecosystem (community update)

**Comment by liushooter** [comment]
## rebase 

- 添加日报14篇  https://rebase.network/posts/category/web3daily
- 尝试将线下 codingday 在线上举行

**Comment by yanyanho** [comment]
# Dapp-Learning

近两周工作进展

## sharing meeting



- 从DC网络刀NYM网络
  - https://www.youtube.com/watch?v=O4j1W8uh_hE&
  - https://www.youtube.com/watch?v=5gbPAwyP8Z0
- 攻击事件分析: Build Finance 7号&8号提案
  - https://www.youtube.com/watch?v=y5TGrGf8vA0&t=14s
- Curve如何成为defi龙头
  - https://www.youtube.com/watch?v=ZlvqdkAizBk

 
## 翻译小组
- MIT公开课翻译第11集进入排版
- MIT公开课翻译第12集进入审稿

## 新建bounty板块
 - https://github.com/Dapp-Learning-DAO/Dapp-Learning/discussions

**Comment by Emilyxiannv** [comment]
### DAOSquare
我们进行了一次关于 IQ Protocol 的主题分享，IQ Protocol 是一个用于数字资产租赁和链上订阅的去中心化货币市场。录播视频地址：https://www.youtube.com/watch?v=QSvpRJzPwPE&list=PLMjOuSK6sA6drvTuFTF30EYqDns3J3Xuh&index=6 ；https://www.youtube.com/watch?v=XvJgNnVpNvE&list=PLMjOuSK6sA6drvTuFTF30EYqDns3J3Xuh&index=7

**Comment by Jackieyewang** [comment]
组织了一次高校的组织讨论，商量dao治理模式，讨论了13个议题，包括dao治理，委员会和活动计划等等
